src/Forms/Form/Form.stories.mdx

```diff
@@ -56,11 +56,313 @@ const fields = useMemo(
 	[]
 );
 ```
+### FormFieldRadioButton
+- A group of radio buttons allows users to select a single item from a list of possible options.
+- All possible options are exposed up front for comparison.
+- Users can only make a single selection from the list of possible options.
+- **Playground**: https://simpleviewinc.github.io/sv-mosaic/?path=/story/formfields-formfieldradio--playground
 
-	
+#### Props (Input Settings)
+* **options** - `array` of `object` optional - Predefined set of mutually exclusive options.
+	* **label** - `string` - Description of the option.
+	* **value** - `string` - Value of the option, this is stored in the Form state.
+
+#### How to use in a form?
+```ts
+const fields = useMemo(
+		() =>
+			[
+				{
+					name: "radio",
+					label: "Radio button label",
+					type: "radio",
+					required: true,
+					disabled: false,
+					inputSettings: {
+						options: [
+							{
+								label: "Label 1",
+								value: "label_1",
+							},
+							{
+								label: "Label 2",
+								value: "label_2",
+							},
+							{
+								label: "Label 3",
+								value: "label_3",
+							},
+						]
+					},
+					helperText: 'Helper text',
+					instructionText: 'Instruction text'
+				}
+			] as FieldDef<FormFieldRadioDef>[],
+		[]
+	);
+```
+
+### FormFieldText
+- Text input fields are used to enter text information, numbers, e-mail addresses, or passwords. 
+- A text field consists of a label and a line with variations in width.
+- **Playground**: https://simpleviewinc.github.io/sv-mosaic/?path=/story/formfields-formfieldtext--playground
+
+#### Props (Input Settings)
+
+* **htmlFor** - `string` optional - Specifies which form element a label is bound to.
+* **maxCharacters** - `number` optional - Defines the limit of characters that are allow to type in the input element.
+* **multiline** - `boolean` optional - When is enabled the text field will expand its height.
+* **placeholder** - `string` optional - The hint displayed in the input before the user enters a value.
+* **prefixElement** - `JSX.Element` optional - Icon at the beginning of the text field.
+* **size** - `enum { xs = '100px', sm = 280px, md = "450px", lg = "620px" }` required - The size of the component.

```

Comment:
For the size please use the following text: "Size of the field. Could be either one of the predefined sizes (please see Sizes section), or a custom size. This property can be internally overwritten if the field is placed inside of a row with other fields." and the following type: "Size | string". There will be a Sizes section to explain more about them, also this way if sizes change we won't need to update it everywhere.